earth_enterprise/src/scons/getversion.py

```diff
@@ -53,10 +63,22 @@ def _GitGeneratedLongVersion():
         return _VersionFromTagHistory(raw)
 
 
+def _IsFirstParentGitDescribeSupported():
+    """Checks whether --first-parent parameter is valid for the
+    version of git available"""
+
+    return opengee.version.is_version_ge(git.Git().version_info, (1, 8, 4))
```

Comment:
I think this is a (relatively) bad way of checking for a feature.  E.g., maybe it's not likely, but what would happen, if someone decided to backport `--first-parent` to version 1.8.3?  Or, if version 2.7.99 dropped `--first-parent`?  You'll have to be reading a lot of release notes, and updating this code.

I think it would be better, if you could run something like `git describe --first-parent --tags` and check for "error: unknown option \`--first-parent'".  Or, if you don't have a Git repository to run this test on, scan the output of `git describe --help` for `--first-parent`.

Reply:
This was my first approach, but somehow I convinced myself to go the other way --- after the ugly import and extra code involved, I'm coming back to this implementation.